telemetry/csharp/ToolkitTelemetryGenerator/Utils/MetricTypeExtensionMethods.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using ToolkitTelemetryGenerator.Models;
+
+namespace ToolkitTelemetryGenerator.Utils
+{
+    public static class MetricTypeExtensionMethods
+    {
+        static readonly Dictionary<string, System.Type> AliasedTypes = new Dictionary<string, System.Type>()
+        {
+            // See https://github.com/aws/aws-toolkit-common/blob/master/telemetry/telemetrySchema.json
+            // properties/types/items/properties/type
+            {"int", typeof(int) },
+            {"double", typeof(double) },
+            {"string", typeof(string) },
+            {"boolean", typeof(bool) },
+        };
+
+        public static bool IsAliasedType(this MetricType type)
```

Comment:
Why is this called aliasedtype? I don't get what this is from the name. Maybe datumType or something like that would be better?

Reply:
The intent is that the MetricType is represented in generated code by a POCO type

Reply:
But the name does not make any sense